src/components/TodoList/TodoList.tsx

```diff
@@ -1,4 +1,15 @@
-// Add the required types and props
-export const TodoList = () => (
-  <>TodoList markup</>
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import { TodoInfo } from '../TodoInfo';
+
+type Props = { 
+  todos: Todo[],
+}
+
+export const TodoList: React.FC<Props> = ({ todos }) => (
+    <section className='TodoList'>
+      {todos.map(todo => (
+        <TodoInfo todo={todo}/>
```

Comment:
don't forget about key by mapping the list 
```suggestion
        <TodoInfo todo={todo} key={todo.id} />
```